Lib9c/Action/MigrateMonsterCollection.cs

```diff
@@ -36,6 +36,11 @@ public override void LoadPlainValue(IValue plainValue)
         public override IAccountStateDelta Execute(IActionContext context)
         {
             var states = context.PreviousStates;
+            if (states.TryGetStakeState(context.Signer, out StakeState _))
+            {
+                return states;
```

Comment:
i think, it will be better throw exception.

Reply:
I made it throw `InvalidOperationException` in [3292248](https://github.com/planetarium/lib9c/pull/983/commits/3292248f099fd3a6a45702de701b5bf9d51c688e)